csharp/Program.cs

```diff
@@ -2,11 +2,48 @@
 
 namespace csharp
 {
+    class Test
+    {
+        public string getIssuer(long n)
```

Comment:
`n` as a variable name could be improved. Avoid using one-letter variable names as much as possible. Variable names should briefly describe what kind of value it holds and/or how it should be used